kong/conf_loader.lua

```diff
@@ -106,13 +106,16 @@ local CONF_INFERENCES = {
 
   database = { enum = { "postgres", "cassandra" }  },
   pg_port = { typ = "number" },
+  pg_database = { typ = "string" },
+  pg_schema = { typ = "string" },
   pg_timeout = { typ = "number" },
   pg_password = { typ = "string" },
   pg_ssl = { typ = "boolean" },
   pg_ssl_verify = { typ = "boolean" },
 
   cassandra_contact_points = { typ = "array" },
   cassandra_port = { typ = "number" },
+  cassandra_keyspace = { typ = "string" },
```

Comment:
`string` configuration values were actually left out intentionally. We only specify `string` for some configuration properties when we want to avoid the configuration parser from inferring the value as a number. For example `nginx_worker_processes`. It's cannot hurt to be more specific, but it increases this commit's diff unecessarily.

Reply:
Oh — are unknown entries (i.e. typos) ignored then? I never noticed it!